packages/orchestration/src/examples/sendAnywhere.contract.js

```diff
@@ -77,7 +67,7 @@ export const start = async (zcf, privateArgs, baggage) => {
       const { chainName, destAddr } = offerArgs;
       const { give } = seat.getProposal();
       const [[kw, amt]] = entries(give);
-      const { denom } = await findBrandInVBank(amt.brand);
+      const { denom } = await V.when(agNamesTools.findBrandInVBank(amt.brand));
```

Comment:
this should come from context, not the closure. Also then it doesn't need `V.when()`

Reply:
Moved to context. It seems like `V.when()` is still needed until `orchestrate` uses `asyncFlow` (like on `dt/af-spike`)